linear_accel_offset:    {'x': 0.025, 'y': -0.11, 'z': 0.45}
angular_vel_offset:     {'x': 0,    'y': 0,     'z': 0}
magnetic_field_offset:  {'x': 0,    'y': 0,     'z': 0}
